vectorstore.go

```diff
@@ -39,6 +40,29 @@ func (vs *LocalVectorStore) LoadJSON(ctx context.Context, filename string) error
 	return vs.Insert(ctx, chunkMap)
 }
 
+// StoreJSON will serialize the `LocalVectorStore` to disk based on the provided filename.
+func (vs *LocalVectorStore) StoreJSON(filename string) error {
+	var chunks []*Chunk
+
+	for _, chunk := range vs.chunks {
+		for _, c := range chunk {
+			chunks = append(chunks, c)
+		}
```

Comment:
This is fine. How about `chunks = append(chunks, chunk...)`?

Reply:
Great suggestion.